test/conftest.py

```diff
@@ -40,4 +41,26 @@ def _mock_assets_dir(env: str) -> str:
     monkeypatch.setattr(SEMConfig, "xfalign_loc", command_loc("xfalign"))
     monkeypatch.setattr(SEMConfig, "xftoxg_loc", command_loc("xftoxg"))
     monkeypatch.setattr(SEMConfig, "convert_loc", command_loc("convert"))
-    monkeypatch.setattr(Config, "bioformats2raw", command_loc("bioformats2raw"))
+    # monkeypatch.setattr(Config, "bioformats2raw", command_loc("bioformats2raw"))
+
+
+@pytest.fixture
+def mock_callback_data(monkeypatch, tmp_path):
+    """
+    In order to assert that callback is appropriate, the callback output
+    is dumped into a temporary file. This file can be json.loaded to get
+    the callback output for tests.
+    """
+    from em_workflows.utils.utils import json as utils_json
+    from json import dumps as real_dumps
```

Comment:
@philipmac @blowekamp 

The `json.dumps` is the last dumps command run in the utils.py (which is called by `send_callback_body`) method from all the workflows.

This was the easiest way to access the dumped data for callback. I wonder if we could mock something else.

Downside: all `json.dumps` in the utils.py is mocked.